lib/components/steps/Home.tsx

```diff
@@ -1,104 +1,71 @@
-import { Box, Button, Typography, Chip } from '@mui/material';
-import { useCdeContext } from "../../CdeContext.tsx";
-import { STEPS } from "../../models.ts";
-import { validateInputMappings, processInputMappings } from '../../services/mappingService.ts';
-import { ModalLayout } from "../layout/ModalLayout.tsx";
-import { StyledTable } from '../common/StyledTable.tsx';
-import { CircleChipDefault, CircleChipSuccess } from '../../icons/index.tsx';
-import { vars } from '../../theme/variables.ts';
+import {Box, Button, Typography, Chip} from '@mui/material';
+import {useCdeContext} from "../../CdeContext.tsx";
+import {StyledTable} from '../common/StyledTable.tsx';
+import {CircleChipDefault, CircleChipSuccess} from '../../icons/index.tsx';
+import {vars} from '../../theme/variables.ts';
+import {STEPS} from "../../models.ts";
 
-const { primary600, gray600, drodownDetailBg } = vars;
+const {primary600, gray600, drodownDetailBg} = vars;
 
 function Home() {
     const {
         setStep,
-        inputMappings,
-        setLoadingMessage,
-        setErrorMessage,
-        setMapping
+        datasetSample,
     } = useCdeContext();
 
-    const handleStartMapping = async () => {
-        if (inputMappings.length == 0) {
-            setErrorMessage('Mapping file not found')
-            return
-        }
-
-        setLoadingMessage('Validating and processing file...');
-
-        try {
-            const isValid = validateInputMappings(inputMappings);
-            if (isValid) {
-                const mapping = processInputMappings(inputMappings);
-                setMapping(mapping);
-                setStep(STEPS.REPOSITORY);
-                console.log(mapping)
-            } else {
-                setErrorMessage('Invalid CSV file format.');
-            }
-        }  catch (error) {
-            if (error instanceof Error) {
-                setErrorMessage(error.message);
-            } else {
-                setErrorMessage('An unknown error occurred');
-            }
-        }
-
-        setLoadingMessage(null);
-    };
 
     return (
-        <ModalLayout>
-            <Box display='flex' alignItems='center' flexDirection='column' px={3} py={6} sx={{
-                background: drodownDetailBg
+        <Box display='flex' alignItems='center' flexDirection='column' px={3} py={6} sx={{
```

Comment:
No longer uses the Modal Layout, the Header is now added in the CDE Modal component. The remaining changes are from the linter